app/views/orders/index.html.erb

```diff
@@ -4,10 +4,12 @@
   <table>
     <tr>
       <th>Order Number</th>
+      <th></th>
```

Comment:
I agree the striped class makes sense and we should put the blank header at the end